source/renderer/app/stores/WalletsStore.js

```diff
@@ -1099,6 +1100,14 @@ export default class WalletsStore extends Store {
     }
   }).bind(this);
 
+  @action _getWalletAddress = async (walletId: string) => {
```

Comment:
@yakovkaravelov does this needs to be an action? The method is not mutating any of the store values?

Reply:
@nikolaglumac I've just fixed it, thanks. 👍